### ISSUE TITLE
exports in package.json

### ISSUE THREAD

**Issue by umaranis** [umaranis/svelte-lexical#76]
Adding the newer exports in package.json throws the following error in certain cases:
```
"exports": {
    ".": {
      "svelte": "./dist/index.js"
    }
  },
```

```
[HMR][Svelte] Unrecoverable HMR error in <RichTextComposer>: next update will trigger a full reload
eval @ VM7:1
index-614a2428.js:973 Uncaught TypeError: Cannot read properties of undefined (reading '$$')
    at init (index-614a2428.js:973:44)
    at new Composer (index-614a2428.js:48886:3)
    at create_fragment (RichTextComposer.svelte:163:32)
    at init (chunk-I2YCUE2Q.js?v=a80c6be1:2203:35)
    at new RichTextComposer (RichTextComposer.svelte:60:33)
    at createProxiedComponent (svelte-hooks.js?v=a80c6be1:341:9)
    at new ProxyComponent (proxy.js?v=a80c6be1:242:7)
    at new Proxy<RichTextComposer> (proxy.js?v=a80c6be1:349:11)
    at create_fragment (App.svelte:5:64)
    at init (chunk-I2YCUE2Q.js?v=a80c6be1:2203:35)
    at new App (App.svelte:8:34)
    at createProxiedComponent (svelte-hooks.js?v=a80c6be1:341:9)
    at new ProxyComponent (proxy.js?v=a80c6be1:242:7)
    at new Proxy<App> (proxy.js?v=a80c6be1:349:11)
    at main.ts:6:13
:5173/vite.svg:1 
        
        
       Failed to load resource: the server responded with a status of 404 (Not Found)```

**Comment by umaranis** [comment]
Here is a svelte issue that seems to be talking about the same thing: https://github.com/sveltejs/svelte/issues/7968

The error is perhaps because of `svelte-lexical` package not using the latest version of vite (it is still using rollup).

**Comment by umaranis** [comment]
Reopening #74 as we can live with the warning till this issue is resolved.

**Comment by umaranis** [comment]
resolved with version 0.4.0
